Classes/Common/Helper.php

```diff
@@ -199,6 +200,19 @@ public static function decrypt($encrypted)
         return $decrypted;
     }
 
+    /**
+     * Convert a domain object that is lazy-loaded to the actual instance, i.e.,
+     * unwrap LazyLoadingProxy if necessary.
+     *
+     * @access public
+     */
+    public static function fromLazy($object)
```

Comment:
I don't like to move this into the Helper class as long we do not use it multiple times in the code. If it's only the `Document::getOwner()`, please move it there.